.github/workflows/static-analysis.yaml

```diff
@@ -1,11 +1,63 @@
-name: Sonar Scan
+name: Static Analysis
```

Comment:
Most of this changes to this file go away as I've moved shell check back to the maven build (so it runs in the context of the PR and not on main with access to the Sonar token)